Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfAnimationChannelTarget.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// The index of the node and TRS property that an animation channel targets.
```

Comment:
Same TRS request here.

Reply:
This is a direct copy from khronos gltf schema. I'd rather stay consistent with their documentation